common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -151,6 +153,10 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
   PKMX_WCHAR p;
   int sdmfI;
 
+  if(m_debug_items) {
+    m_debug_items->push_group_enter(gp);
+  }
```

Comment:
Corresponds to:
https://github.com/keymanapp/keyman/blob/75e47c1ccc36e1698bafa8178acf8140f88e4e2b/windows/src/engine/keyman32/kmprocess.cpp#L235